voice-interface/src/main/java/com/derpgroup/derpwizard/voice/model/VoiceInput.java

```diff
@@ -86,7 +86,7 @@
    * 
    * @return The associated metadata, never null
    */
-  @NonNull JSONObject getMetadata();
+  @NonNull CommonMetadata getMetadata();
```

Comment:
Now that JSONObject is removed, we can kill the import at the top of this file:
import net.sf.json.JSONObject;